whenchanged/whenchanged.py

```diff
@@ -30,6 +33,35 @@
     # Standard library
     import subprocess
 
+class CmdRunner():
```

Comment:
Having a class for this seems a bit overkill. Can you find a way to integrate this functionality into the WhenChanged class?